spec/features/batches/sort_requests_spec.rb

```diff
@@ -3,10 +3,11 @@
 require 'rails_helper'
```

Comment:
Not much actually changes in the overall workflow. Just add tests for the message generation and now we click a link, not a button.